Bottom: 5c7b3bee12223d90e488d12107fbb08109b0de70
Top:    6e7e62f469359edd83b1d880a39c91bfdf89fed3
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-11 14:30:55 +0000

Refresh of w32-update-pixman-recipe-to-0

---

diff --git a/w32/recipes/pixman.recipe b/w32/recipes/pixman.recipe
index 73fa26c..efcabed 100644
--- a/w32/recipes/pixman.recipe
+++ b/w32/recipes/pixman.recipe
@@ -1,4 +1,4 @@
 name=pixman
-version=0.16.2
+version=0.20.2
 release=1
 download=http://www.cairographics.org/releases
